manifests/kots-app.yaml

```diff
@@ -7,7 +7,11 @@ spec:
   title: CARTO
   icon: https://upload.wikimedia.org/wikipedia/commons/thumb/1/13/CARTO-logo.svg/1280px-CARTO-logo.svg.png
   allowRollback: false
+  # TODO: Define min supported kots version
+  # targetKotsVersion: "1.60.0"
+  # minKotsVersion: "1.40.0"
```

Comment:
This one was already included: https://github.com/CartoDB/carto-selfhosted-helm/blob/09bc58619852aada014829a4134c6c64f0dcb2dd/manifests/kots-app.yaml#L24

Reply:
```suggestion
```